bigquery/query_property_places.md

```diff
@@ -6,9 +6,11 @@ parent: DC to BQ Sample Queries
 grand_parent: BigQuery
 ---
 
-# Query Category: Properties of Places (and other entities)
+# Query Category: Properties (Statistics/Variables) of (about) Places (and other entities)
```

Comment:
how about "Statistical variables associated with Places (and other entities)" ?

Reply:
Done